src/lint/lint.ts

```diff
@@ -320,8 +320,12 @@ async function fix(
       await applyEdits(edits, makeParseLoader(analyzer, analysis));
 
   for (const [newPath, newContents] of editedFiles) {
-    await fs.writeFile(
-        path.join(config.root, newPath), newContents, {encoding: 'utf8'});
+    const documentPath = urlLoader.getFilePath(newPath);
```

Comment:
`newPath` is already an absolute path, and `urlLoader.getFilePath` returns the same absolute path...I'm a bit confused now: either we use urlLoader to get the relative path here, or we don't use it at all and always print absolute paths. What should be done here?

Reply:
Hm. newPath should be a URL, documentPath should be a FS path. Is that not what you're seeing?